tests/cases/fourslash/getOccurrencesAbstract.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path='fourslash.ts' />
+
+////[|abstract|] class Animal {
+////    [|abstract|] walk(): void;
+////    [|abstract|] makeSound(): void;
+////}
+////// Abstract class below should not get highlighted
+////abstract class Foo {
+////    abstract foo(): void;
+////    abstract bar(): void;
+////}
+
+test.ranges().forEach(r => {

```

Comment:
Will do, didn't actually notice I was calling `test.ranges` that many times. Thanks!